fem/fespace.hpp

```diff
@@ -723,6 +723,12 @@ class QuadratureSpace
    /// Return the total number of quadrature points.
    int GetSize() const { return size; }
 
+   /// Returns the mesh
+   inline Mesh *GetMesh() const { return mesh; }
+
+   /// Returns number of elements in the mesh.
+   inline int GetNE() const { return mesh->GetNE(); }
+
```

Comment:
These should be nice improvements to life functions to ```QuadratureSpace```.

Reply:
I do not understand the comment? I interpret this as positive feadback and not a request for change. Is that correct?
If so, thank you! If not could you eleborate?

Reply:
It was just positive feedback :) and nothing for you to fix.